src/components/orders/OrderDetails/index.tsx

```diff
@@ -24,7 +25,7 @@ export type Props = {
   trades: Trade[]
   isOrderLoading: boolean
   areTradesLoading: boolean
-  errors: Record<string, string>
+  errors: any
```

Comment:
The best thing here I think is to define the error type and when creating it noting whether it's and error or a warning.

The type would be something like

```ts
export type UiError = {
  message: string
  type: 'warn' | 'error'
}
```

And another type here for aggregating all errors. 
Don't remember why exactly it was created as a Record, so we could change it to an Array, but anyway:

```ts
type Errors = Record<string, UiError>
```